core/CronArchive/SegmentArchivingRequestUrlProvider.php

```diff
@@ -138,6 +158,15 @@ private function getCreatedTimeOfSegment($idSite, $segmentDefinition)
                 if ($createdTime->getTimestamp() < $earliestCreatedTime->getTimestamp()) {
                     $earliestCreatedTime = $createdTime;
                 }
+
+                if (!empty($segment['ts_last_edit'])) {
```

Comment:
Or maybe if `ts_last_edit ` is not set we could set it automatically to `ts_created`? Not a big advantage but I'd find it a bit cleaner. up to you...